Pasteboard Viewer/ContentView.swift

```diff
@@ -28,9 +28,14 @@ struct ContentView: View {
 	private func setWindowTitle() {
 		AppDelegate.shared.window.title = selectedType?.title ?? App.name
 	}
+	
```

Comment:
You can make the AppDelegate `selectedType` a `Binding` instead and then only set the binding in the init here from the `@State` (`$selectedType`) and the binding will keep the value up to date itself.